From c3a009b4fb2f0630ac891727810058b48c8e2670 Mon Sep 17 00:00:00 2001
From: Ryan Lortie <desrt@desrt.ca>
Date: Mon, 2 Mar 2015 12:21:31 -0500
Subject: [PATCH 25/29] win32: #include gi18n-lib.h appropriately

...but first make sure we pull in "config.h" to get GETTEXT_PACKAGE
properly defined.

This problem popped up while trying to do cross-builds of gtk+ to
Windows under jhbuild.

https://bugzilla.gnome.org/show_bug.cgi?id=745402
---
 gdk-pixbuf/io-gdip-bmp.c   | 3 +++
 gdk-pixbuf/io-gdip-emf.c   | 3 +++
 gdk-pixbuf/io-gdip-gif.c   | 3 +++
 gdk-pixbuf/io-gdip-ico.c   | 3 +++
 gdk-pixbuf/io-gdip-jpeg.c  | 3 +++
 gdk-pixbuf/io-gdip-tiff.c  | 3 +++
 gdk-pixbuf/io-gdip-utils.c | 3 +++
 gdk-pixbuf/io-gdip-wmf.c   | 3 +++
 8 files changed, 24 insertions(+)

diff --git a/gdk-pixbuf/io-gdip-bmp.c b/gdk-pixbuf/io-gdip-bmp.c
index 0eee035..420af7c 100644
--- a/gdk-pixbuf/io-gdip-bmp.c
+++ b/gdk-pixbuf/io-gdip-bmp.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 static gboolean
diff --git a/gdk-pixbuf/io-gdip-emf.c b/gdk-pixbuf/io-gdip-emf.c
index edba045..77e417d 100644
--- a/gdk-pixbuf/io-gdip-emf.c
+++ b/gdk-pixbuf/io-gdip-emf.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 #ifndef INCLUDE_gdiplus
diff --git a/gdk-pixbuf/io-gdip-gif.c b/gdk-pixbuf/io-gdip-gif.c
index 4d4686f..68d49bf 100644
--- a/gdk-pixbuf/io-gdip-gif.c
+++ b/gdk-pixbuf/io-gdip-gif.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 static gboolean
diff --git a/gdk-pixbuf/io-gdip-ico.c b/gdk-pixbuf/io-gdip-ico.c
index 1403780..f17fbf9 100644
--- a/gdk-pixbuf/io-gdip-ico.c
+++ b/gdk-pixbuf/io-gdip-ico.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 #ifndef INCLUDE_gdiplus
diff --git a/gdk-pixbuf/io-gdip-jpeg.c b/gdk-pixbuf/io-gdip-jpeg.c
index bc320e2..0f2db1d 100644
--- a/gdk-pixbuf/io-gdip-jpeg.c
+++ b/gdk-pixbuf/io-gdip-jpeg.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #define INITGUID
 #include "io-gdip-utils.h"
 
diff --git a/gdk-pixbuf/io-gdip-tiff.c b/gdk-pixbuf/io-gdip-tiff.c
index 6c81e65..100f654 100644
--- a/gdk-pixbuf/io-gdip-tiff.c
+++ b/gdk-pixbuf/io-gdip-tiff.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 static gboolean
diff --git a/gdk-pixbuf/io-gdip-utils.c b/gdk-pixbuf/io-gdip-utils.c
index 5a31c75..e5f3e14 100644
--- a/gdk-pixbuf/io-gdip-utils.c
+++ b/gdk-pixbuf/io-gdip-utils.c
@@ -20,6 +20,9 @@
  */
 
 #define INITGUID
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include <ole2.h>
 
 #include "io-gdip-utils.h"
diff --git a/gdk-pixbuf/io-gdip-wmf.c b/gdk-pixbuf/io-gdip-wmf.c
index cfe580c..c438c89 100644
--- a/gdk-pixbuf/io-gdip-wmf.c
+++ b/gdk-pixbuf/io-gdip-wmf.c
@@ -19,6 +19,9 @@
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "config.h"
+#include <glib/gi18n-lib.h>
+
 #include "io-gdip-utils.h"
 
 #ifndef INCLUDE_gdiplus
-- 
2.1.0

